pyirf/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.1.0-alpha'
\ No newline at end of file
+__version__ = "0.1.0-dev"
```

Comment:
- not allowed in python versions, see https://www.python.org/dev/peps/pep-0440/#public-version-identifiers

Reply:
should this version be something like 0.1aN.devM ?
It is not clear to me what N and M are, though

Reply:
These are counters, you can have multiple alpha versions, like 0.1.0a1, 0.1.0a2, but like I said, just leave it at 0.1.0 for now, it does not matter.

Reply:
OK

Reply:
Ah, side-question: I had problems building the docs on `readthedocs` when reading the version from `pyirf.version` because the installation of the package would fail.
I would like to fix this later, so for the moment I hard-coded the version in the docs to be the same as in version (I know, it's not a good thing...).

I was thinking though, can I copy-paste the lines used also in setup.py to open the file instead of importing it?
That way we still wouldn't need to install `pyirf` to build the docs and we can use the requirements file.

Reply:
I think the only problem there was that you need to move the `sys.path.insert` line before importing pyirf